csharp/src/Microsoft.ML.OnnxRuntime/NativeMethods.cs

```diff
@@ -127,6 +127,15 @@ public struct OrtApi
         public IntPtr ReleaseTensorTypeAndShapeInfo;
         public IntPtr ReleaseSessionOptions;
         public IntPtr ReleaseCustomOpDomain;
+        public IntPtr GetDenotationFromTypeInfo;
+        public IntPtr CastTypeInfoToMapTypeInfo;
+        public IntPtr CastTypeInfoToSequenceTypeInfo;
+        public IntPtr GetMapKeyType;
+        public IntPtr GetMapValueType;
+        public IntPtr GetSequenceElementType;
+        public IntPtr ReleaseMapTypeInfo;
+        public IntPtr ReleaseSequenceTypeInfo;
+        public IntPtr SessionEndProfiling;
```

Comment:
Need to support SessionEndProfiling - so we need to bring in everything until this API to mirror the C struct. The others don't need their corresponding delegates.